src/test/java/com/commercetools/project/sync/util/TestUtils.java

```diff
@@ -46,9 +46,12 @@ public static void assertAllSyncersLoggingEvents(
     assertProductSyncerLoggingEvents(syncerTestLogger, numberOfResources);
     assertInventoryEntrySyncerLoggingEvents(syncerTestLogger, numberOfResources);
     assertCartDiscountSyncerLoggingEvents(syncerTestLogger, numberOfResources);
+    assertStateSyncerLoggingEvents(
+        syncerTestLogger,
+        numberOfResources + 1); // 2 states as 1 state is built-in and it cant be deleted
 
     // Every sync module (6 modules) is expected to have 2 logs (start and stats summary) = 12
```

Comment:
is this comment still valid? Just to cross check

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/133/commits/446caae5aacd2064b4580790e3c944b97110e27b